parm/config/config.resources.emc.dyn

```diff
@@ -43,7 +43,9 @@ if [ $step = "prep" -o $step = "prepbufr" ]; then
     eval "export npe_$step=4"
     eval "export npe_node_$step=2"
     eval "export nth_$step=1"
-    eval "export memory_$step=40G"
+    if [[ "$machine" != "WCOSS2" ]]; then
+      eval "export memory_$step=40G"
+    fi
```

Comment:
What value for memory is being given on WCOSS2?
Not sure if this warrants an `if-block`

Reply:
The prep job runs exclusively on one node on WCOSS2 so we don't want a memory setting for it in here (setup scripts only set exclhost in the xml if no memory is set for job). The prep job fails with 40G on WCOSS2 when tested, 40G is the R&D value that has been tested.

Reply:
Ok. thanks.  
In a subsequent PR, we might want to know what the memory footprint is on WCOSS2